moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -397,7 +397,16 @@ KinematicState Servo::getNextJointState(const ServoInput& command)
   robot_state->copyJointGroupPositions(joint_model_group, current_state.positions);
   robot_state->copyJointGroupVelocities(joint_model_group, current_state.velocities);
 
+  // If we provide a previous state for drift correction, set the robot position to that.
+  auto initial_state = current_state;
+  if (servo_params_.apply_anti_drift && last_commanded_state_.has_value())
+  {
+    robot_state->setJointGroupPositions(joint_model_group, last_commanded_state_.value().positions);
```

Comment:
Since we know last_commanded_state_ has a value

```suggestion
    robot_state->setJointGroupPositions(joint_model_group, *last_commanded_state_.positions);
```